src/types/aws/ap/index.ts

```diff
@@ -29,7 +29,7 @@ type EndowmentBase = {
 export type EndowmentProfile = EndowmentBase & {
   contact_email: string;
   logo: string;
```

Comment:
Field `logo` is not marked as optional since it will have a default value.
For more details see CU task [Add placeholder logo upon creation of new AST](https://app.clickup.com/t/865c65bug).